src/report/NodeDensityReport.java

```diff
@@ -57,6 +57,9 @@
 	//========================================================================//
 	// Instance vars
 	//========================================================================//
+    /** Layer selection. Use -1 for merging all layers into a single report. */
+	protected int layer = -1;
```

Comment:
with merging into "one file", do you mean having all the HeatMaps of the upper Floors casted to the groundfloor?

Reply:
Exactly